### ISSUE TITLE
Using console.log instead of something else?

### ISSUE THREAD

**Issue by cdwiegand** [balderdashy/sails-mysql#118]
Am writing an API to use sails-mysql (thanks, btw, for making this!). But since findOrCreate wasn't working for me, I ended up just calling create() and ignoring on my side if error.code == 'ER_DUP_ENTRY', since that's ok (I'm just logging information, I'm not using the resultant record). The problem is that console gets filled up with the errors, though, and I was thinking that that should be disable-able, as that's not always desirable output (even if it's only on a web-server). I should be able to suppress that so as not to fill up some webserver log somewhere.

I'm willing to fork and try my hand at it, if this sounds reasonable to others.

**Comment by sailsbot** [comment]
Thanks for posting, @cdwiegand.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
